src/components/Board.re

```diff
@@ -5,11 +5,11 @@ module Card = {
   let make = (~value, _children) => {
     ...component,
     render: (_self) =>
-      if (value == 0) {
-        <div className="card" />
-      } else {
-        <div className=("card card_number_" ++ string_of_int(value))>
-          (ReasonReact.stringToElement(string_of_int(value)))
+      switch value {
+      | 0 => <div className="card" />
+      | value =>
+        <div className={j|card card_number_$value|j}>
```

Comment:
This is https://bucklescript.github.io/bucklescript/Manual.html#_unicode_support_with_string_interpolation_since_1_7_0

Also removes the need of bringing in the `Pervasives` module in the js output